### PR TITLE
zh: add openapi doc

### PR DESCRIPTION
**Pull request by Ehco1996**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

v2.1.0

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/dm/issues/1982
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/open-api.md [hunk 1]
```diff
@@ -0,0 +1,1071 @@
+---
+title: DM OpenAPI
+summary: 了解如何使用 OpenAPI 接口来管理集群状态和数据同步。
+aliases: ['/docs-cn/tidb-data-migration/dev/open-api/']
```
Comment (Liuxiaozhen12) [inline zh/open-api.md:4]:
```suggestion
```
A new document does not need aliases.

Comment (Ehco1996) [reply zh/open-api.md:4]: good catch

File: zh/open-api.md [hunk 2]
```diff
@@ -0,0 +1,1070 @@
+---
+title: DM OpenAPI
+summary: 了解如何使用 OpenAPI 接口来管理集群状态和数据同步。
+---
+
+# DM OpenAPI
+
+> **注意：**
+>
+> 本文描述特性仍为实验特性，不建议在生产环境下使用 。
```
Comment (Liuxiaozhen12) [inline zh/open-api.md:10]:
```suggestion
> **警告：**
>
> 当前该功能为实验特性，不建议在生产环境中使用。
```

File: zh/open-api.md [hunk 3]
```diff
@@ -0,0 +1,1070 @@
...
+summary: 了解如何使用 OpenAPI 接口来管理集群状态和数据同步。
+---
+
+# DM OpenAPI
+
+> **警告：**
+>
+> 当前该功能为实验特性，不建议在生产环境中使用。
+
+DM 提供 OpenAPI 功能，用户可通过 OpenAPI 对 DM 集群进行查询和运维操作。OpenAPI 的总体功能和 [dmctl工具](./dmctl-introduction.md)类似。
```
Comment (GMHDBJD) [inline zh/open-api.md:12]:
```suggestion
DM 提供 OpenAPI 功能，用户可通过 OpenAPI 对 DM 集群进行查询和运维操作。OpenAPI 的总体功能和 [dmctl 工具](./dmctl-introduction.md)类似。
```

File: zh/open-api.md [hunk 4]
```diff
@@ -0,0 +1,1070 @@
...
+
+> **注意：**
+>
+> DM 提供符合 OpenAPI 3.0.0 标准的 [Spec 文档](https://github.com/pingcap/dm/blob/master/openapi/spec/dm.yaml)，其中包含了所有 API 的请求参数和返回体，你可自行复制到如 [Swagger Editor](https://editor.swagger.io/) 等工具中在线预览文档。
```
Comment (GMHDBJD) [inline zh/open-api.md:16]: now should be ticdc?

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Comment by Liuxiaozhen12** [comment]
@Ehco1996 Could you please add some descriptions for the parameters that may not so familiar to the readers? Of course, it depends on your judgement and time...

**Comment by Liuxiaozhen12** [comment]
> @Ehco1996 Could you please add some descriptions for the parameters that may not so familiar to the readers? Of course, it depends on your judgement and time...

the spec docs has already contained all the descriptions.

**Review by Liuxiaozhen12 — APPROVED** [review]

File: zh/open-api.md [hunk 5]
```diff
@@ -0,0 +1,1077 @@
...
+对 API 发起的请求后，如发生错误，返回错误信息的格式如下所示：
+
+```json
+{
+    "error_msg": "",
+    "error_code": ""
+}
+```
+
+如上所示， `error_msg` 描述错误信息， `error_code` 则是对应的错误码。
```
Comment (Liuxiaozhen12) [inline zh/open-api.md:75]:
```suggestion
如上所示，`error_msg` 描述错误信息， `error_code` 则是对应的错误码。
```

Comment (Liuxiaozhen12) [inline zh/open-api.md]:
```suggestion
如上所示， `error_msg` 描述错误信息，`error_code` 则是对应的错误码。
```

File: zh/open-api.md [hunk 6]
```diff
@@ -0,0 +1,1077 @@
...
+对 API 发起的请求后，如发生错误，返回错误信息的格式如下所示：
+
+```json
+{
+    "error_msg": "",
+    "error_code": ""
+}
+```
+
+如上所示，`error_msg` 描述错误信息， `error_code` 则是对应的错误码。
```
Comment (Liuxiaozhen12) [inline zh/open-api.md:75]:
```suggestion
如上所示，`error_msg` 描述错误信息，`error_code` 则是对应的错误码。
```

**Pull request merged**